apps/api/pages/api/event-types/_post.ts

```diff
@@ -284,8 +284,8 @@ async function postHandler(req: NextApiRequest) {
   await checkPermissions(req);
 
   if (parsedBody.parentId) {
-    await checkParentEventOwnership(parsedBody.parentId, userId);
-    await checkUserMembership(parsedBody.parentId, parsedBody.userId);
+    await checkParentEventOwnership(req);
+    await checkUserMembership(req);
```

Comment:
We could also pass parsedBody as down below but I like the self-containment here. https://github.com/calcom/cal.com/pull/11867/files#diff-53284e5334f4b512fe926bb35af4b182fc262e5a2a0dda8ac2f6c1d418163b86R295

Reply:
Yeah, I agree. This is better 🙏